src/content/configuration/module.mdx

```diff
@@ -249,6 +249,105 @@ module.exports = {
 }
 ```
 
+If `Rule.type` is an `asset` then `Rules.parser` option may be an object or function that describes a condition whether to encode file contents to Base64 or emit it as a separate file into the output directory.
+
+If `Rule.type` is an `asset` or `asset/inline` then `Rule.generator` option may be an object that describes an encoding of the module source or a function that encodes a module source by a custom algorithm.
+
+For more info - see [Asset Modules](/guides/asset-modules/)
+
+## `Rule.parse.dataUrlCondition` as an object
+
+```js
+module.exports = {
+  //...
+  module: {
+    rules: [
+      {
+        //...
+        parser: {
+          dataUrlCondition: {
+            maxSize: 8096
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+## `dataUrlCondition.minSize`
+
+`number`
+
+If a module source size is less than `maxSize` then module will be injected into the bundle as a Base64-encoded string, otherwise module file will be emitted into the output directory.
+
+## `Rule.parse.dataUrlCondition` as a function
+
+`function(source, { filename, module }) => boolean`
+
+If the function returns true, then module will be injected into the bundle as base64-encoded string, otherwise module file will be emitted info output directory.
```

Comment:
```suggestion
If the function returns true, then module will be injected into the bundle as Base64-encoded string, otherwise module file will be emitted info output directory.
```